index.html

```diff
@@ -0,0 +1,216 @@
+            <li>We attend church each Sunday as a family and uphold the sacredness of the Sabbath.</li>
+            <li>We vow to follow the Ten Commandments, trusting in God to guide our life.</li>
+        </ul>
+
+        <h2>Article II: Family and Fellowship</h2>
+        <ul class="principles">
+            <li>We acknowledge the blessing of family. We shall appreciate and cherish our immediate and extended family, and the time we have with them.</li>
+        </ul>
+
+        <h2>Article III: Financial Security and Stability</h2>
+        <ul class="principles">
+            <li>We shall pray to obtain financial security, stability, and independence. We shall live free from financial debt
+                and manage our resources wisely.</li>
+            <li>We would like to retire safely and comfortably, without financial strain.</li>
+            <li>We would like to take one major vacation each year, in addition to zero to several smaller, domestic trips—living magical memories.</li>
+        </ul>
+
+        <h2>Article IV: Health and Well-Being</h2>
+        <ul class="principles">
+            <li>We shall hope to overflow with love for one another in our marriage.
+            </li>
+            <li>We shall attempt to eat a pure vegan diet and maintain good health, acknowledging the positive impact on our health and well-being.</li>
+            <li>We shall attempt to build happpiness in our home.</li>
+        </ul>
+
+        <h2>Article V: Work and Balance</h2>
+        <ul class="principles">
+            <li>We shall seek to live in the moment, with the spirit of God, enjoying our life and obtaining balance between work and life.</li>
+            <li>Our work will not only be a source of income but also be fulfilling to us.</li>
+            <li>We shall be very organized in our personal and work affairs.</li>
+            <li>Jacob shall attempt to obtain a Master's Degree.</li>
+            <li>Taylor shall attempt to obtain a Bachelor's Degree.</li>
+        </ul>
+
+        <h2>Article VI: Home and Family Life</h2>
+        <ul class="principles">
+            <li>We pray to live in a safe neighborhood and in a house we are pleased with.</li>
+            <li>We pray to have at least one child before Taylor reaches the age of thirty.</li>
+            <li>We would like to achieve our goal body composition.</li>
+            <li>We would like to take two walks or hikes per week for health and vitality—Jacob shall also attempt to run at least three times per week.</li>
+        </ul>
+
+        <h2>Article VII: Reading</h2>
+        <ul class="principles">
+            <li>We commit to reading the entire Bible, deepening our union with God's word.</li>
+            <li>Jacob commits to reading "The 7 Habits of Highly Effective People".</li>
+        </ul>
+
+        <h2>Amendment I: Removed Provisions</h2>
+        <ul>
+            <li>We shall no longer pursue the running of a marathon.</li>
```

Comment:
can we still run the turkey trot one year?

Reply:
Awh, I will add that

Reply:
<img width="658" alt="image" src="https://github.com/user-attachments/assets/fb6343b5-4c7b-4b37-9609-45eaee8737ee">